python_openapi_client/fatcat_openapi_client/__init__.py

```diff
@@ -13,7 +11,6 @@
 """
 
 
-from __future__ import absolute_import
```

Comment:
All of the code under `python_openapi_client/` is code-generated. I do a couple dirty hacks for some of the code generation (eg, rust code) to make some tweaks, but I am generally disposed to just leave the code-generated stuff as-is as much as possible, instead of layering another set of transforms on top.

The code generation tool we use is pushing a major release in the next month or so, we can check if the output has been updated to remove this kind of backwards compatibility when that comes out. https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/python.md

Reply:
`git checkout master -- python_openapi_client`